src/shims/foreign_items.rs

```diff
@@ -45,7 +45,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         // List taken from `libstd/sys_common/alloc.rs`.
```

Comment:
```suggestion
        // List taken from `library/std/src/sys/common/alloc.rs`.
```
Looks like this file changed location.